app/com/gu/identity/frontend/configuration/Configuration.scala

```
@@ -0,0 +1,24 @@
+package com.gu.identity.frontend.configuration
+
+import com.google.inject.ImplementedBy
+import play.api.Play.current
+
+
+@ImplementedBy(classOf[ApplicationConfiguration])
+trait Configuration {
+  val identityCookieDomain: String
+
+  val identityApiKey: String
+  val identityApiHost: String
+}
+
+class ApplicationConfiguration extends Configuration {
+
+  private def getString(path: String) =
+    current.configuration.getString(path).getOrElse(sys.error(s"Missing configuration: $path"))
+
+  val identityCookieDomain = getString("identity.frontend.cookieDomain")
+
+  val identityApiKey = getString("identity.api.key")
```

It would be nice to fail fast at boot if the configuration is invalid or has missing keys